### ISSUE TITLE
Content of 'extra' is not saved to note (ETH Save item details to note)

### ISSUE THREAD

**Issue by bjohas** [edtechhub/zotero-edtechhub#60]
It woud be good if the content of the 'extra' field was saved to the note when using the menu item "ETH Save item details to note". As there may not be a space for it in RIS, the contents of extra could just be written to the note after ER-. 

(The merge automatically does a 'save items details'  - the above should also be done in conjunction with the merge operation.)

Finally, it would also be good to write out 'date added' and 'date modified', both for "ETH Save item details to note", as well as for the merge.

Also see https://github.com/edtechhub/zotero-edtechhub/issues/64.

**Comment by retorquere** [comment]
We currently present the full list of items to the translator and have it do its thing. To do this we'd either need to export the items one by one so we can postfix them, or break them apart after export to insert the `extra` contents.

**Comment by bjohas** [comment]
How about the 2nd one of these? We just let the translator work, as before.

Then, we are already some extra fields at the end of the note, where 'itemkey' and 'itemkeyold' are added
```
group:2259720
itemKey:AANVMIFC
itemKeyOld:BF3U4PX7
```
The extra fields could just be output after that? 

It's not important to me to see which extra belongs to each item, or have them structured in some way.

This layout:
```
group:2259720
itemKey:AANVMIFC
itemKeyOld:BF3U4PX7
Extra: ...
ExtraOld: ...
```
would be fine.

**Comment by retorquere** [comment]
Oh, if you don't need to know which items they cam from, that's easy

**Comment by bjohas** [comment]
Just to confirm: We don't need to know where they come from - so that's good!

**Comment by retorquere** [comment]
You want to keep this for pair programming? It's an easy change. Might be a good break-in session.

**Comment by bjohas** [comment]
Great!

(For the pair-programming, it would also be good to look at how to manage releases.)

**Comment by retorquere** [comment]
I would want to walk through all the essential parts:

* how the release chain works
* the basics of Zotero, specifically the async parts, and finding stuff in the zotero source tree
* how monkey patching works

**Comment by bjohas** [comment]
Great!

**Comment by bjohas** [comment]
Some code in comments was added that nearly fixed this issue.

**Comment by bjohas** [comment]
Fixed in v0.0.32
